tests/_benderjs/ckeditor/static/extensions.js

```diff
@@ -185,6 +185,35 @@
 		}
 	};
 
+	/**
+	 * Assertion function which uses {@link bender.editorBot.combo}, apply style provided by this combo and compare result.
```

Comment:
Maybe "[…] apply style provided by the passed combo value […]"?

Reply:
I'm not sure if this is what you mean. In case that this will be only one problem, then feel free to correct description according to your needs.